src/test/packages.test.ts

```diff
@@ -630,7 +630,7 @@ describe("RockCXXPackage", function () {
                 returns(() => Promise.resolve(executable));
             const expectedCustomDebugConfig: vscode.DebugConfiguration = {
                 type: "cppdbg",
-                name: relative(subject.path, executable),
+                name: relative(subject.info.builddir, executable),
                 request: "launch",
                 program: executable,
```

Comment:
Actually, it would be better to make the executable relative to builddir as well and use `${rock:buildDir}`. This way, one can check-in the launch.json in git and share the definitions.

Reply:
Updated.